modules/virtualisation/containers.nix

```diff
@@ -0,0 +1,89 @@
+{ config, lib, pkgs, ... }:
+let
+  cfg = config.virtualisation.containers;
+
+  inherit (lib) mkOption types;
+
+  toml = pkgs.formats.toml { };
+in {
+  meta.maintainers = [ lib.hm.maintainers.bad ];
```

Comment:
There is no such maintainer listed anywhere, as far as I can see?

Reply:
I think this is a leftover from my original PR, where I was the only maintainer. 
I don't feel confident in being the sole maintainer but I would be happy to become a second maintainer for this module.

Reply:
I'll let you decide on that once the PR is ready @MaeIsBad . Then you can have an idea of what you're getting yourself into :grin: